superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -253,33 +243,31 @@ class TableElement extends React.PureComponent {
 
     const metadata = (
       <div
-        onMouseEnter={() => this.setHover(true)}
-        onMouseLeave={() => this.setHover(false)}
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
         css={{ paddingTop: 6 }}
       >
-        {this.renderWell()}
+        {renderWell()}
         <div>
           {cols &&
             cols.map(col => <ColumnElement column={col} key={col.name} />)}
```

Comment:
This can be simplified to `{cols?.map(col => <ColumnElement column={col} key={col.name} />)}`

Reply:
Sending up a commit!